src/services/fileStorage/model.js

```diff
@@ -1,21 +1,25 @@
 const mongoose = require('mongoose');
+const uuid = require('uuid/v4');
 
 const { Schema } = mongoose;
 
-const permissionSchema = new Schema({
-	refId: {
-		type: Schema.Types.ObjectId,
-		refPath: 'refPermModel',
-	},
-	refPermModel: {
-		type: String,
-		enum: ['user', 'role'],
+const permissionSchema = new Schema(
+	{
+		refId: {
+			type: Schema.Types.ObjectId,
+			refPath: 'refPermModel',
+		},
+		refPermModel: {
+			type: String,
+			enum: ['user', 'role'],
```

Comment:
add target and targetmodel, see news model

Reply:
this belongs to the next task